tests/cross_account/main.tf

```diff
@@ -29,68 +29,16 @@ module "share" {
 
   allow_external_principals = true
 
-  resources = [
```

Comment:
Are you removing these resources because they're not necessary for the cross-account test?  You're simply testing  the sharing of a ram share even though it's empty?  I just wanted to check if I'm understanding things correctly.

Reply:
That's correct. The other test exercises the sharing of the resource already.